code/game/objects/items/circuitboards/computer_circuitboards.dm

```diff
@@ -209,6 +209,12 @@
 	name = "Supply Console (Computer Board)"
 	build_path = /obj/machinery/computer/cargo
 	var/contraband = FALSE
+	var/space_gear = FALSE
+	var/heavy_firearms = FALSE
+	var/medco_trade = FALSE
+	var/blackmarket = FALSE
+	var/animal_handing = FALSE
+	var/adv_sci = FALSE
```

Comment:
Excuse me, can't you just use one damn bitfield variable instead of all these booleans?